### ISSUE TITLE
Question_ids for VQA tsv file

### ISSUE THREAD

**Issue by cdancette** [OFA-Sys/OFA#188]
It seems that the VQA validation tsv file contains only image-id and not question-ids (the first two values of the tsv are always equal).

```bash
$ cat vqa-ofa/vqa_val.tsv | cut -f 1-4 | head
101096  101096  what color are the letters of the stop sign?    1.0|!+white
101096  101096  are there any cars on the road? 1.0|!+no
101096  101096  what is the name of the street running along the left side of the photo?        1.0|!+pacific
221282  221282  what are they cutting?  1.0|!+pizza
221282  221282  are the lights on in this room? 1.0|!+yes
```

Could you provide the file with question-id and image-id? Or the script that you used to create this tsv so that I can regenerate the correct one ?

Thank you very much

**Comment by yangapku** [comment]
Hi, yes this is indeed a preprocessing mistake existing in the training & valid dataset part. Could you please have a look at this issue #173 and see if it can help?

**Comment by cdancette** [comment]
Thank you, by comparing with the original annotations I was able to recover the original question ids.

**Comment by yangapku** [comment]
Great! Would you mind to share a mapping (like mapping the line-id to the question-id) to help fix the problem?

**Comment by cdancette** [comment]
Here is for the validation set

https://drive.google.com/file/d/1BNMIGMIU11zpOL7IahmRSeKC6rYB0nfw/view?usp=sharing

I will send you the one for the training set as well when I have it.

**Comment by cdancette** [comment]
Also i noticed that there are a few duplicate image-questions in VQA val, so this file might contain a few that were assigned to the same question_id (i found 5 examples in the file
).
